source/extensions/filters/http/admission_control/evaluators/default_evaluator.cc

```diff
@@ -0,0 +1,68 @@
+#include "extensions/filters/http/admission_control/evaluators/default_evaluator.h"
+
+#include <algorithm>
+
+#include "envoy/grpc/status.h"
+
+#include "common/common/assert.h"
+#include "common/common/enum_to_int.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace AdmissionControl {
+
+DefaultResponseEvaluator::DefaultResponseEvaluator(
+    envoy::extensions::filters::http::admission_control::v3alpha::AdmissionControl::
+        DefaultEvaluationCriteria evaluation_criteria) {
+  // HTTP status.
+  if (evaluation_criteria.http_success_status_size() > 0) {
+    for (const auto& range : evaluation_criteria.http_success_status()) {
+      RELEASE_ASSERT(range.start() <= range.end() && range.start() < 600 && range.start() >= 100 &&
```

Comment:
assuming this is in the configuration path throw an `EnvoyException` instead to propagate the error message back to the user/control plane. `RELEASE_ASSERT` will crash the process